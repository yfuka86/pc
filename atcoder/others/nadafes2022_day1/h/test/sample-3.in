6 4
2 5
3 5
1 4
4 5
4 6
3 1 2 3
3 2 4 5
2 1 6
6 1 2 3 4 5 6
