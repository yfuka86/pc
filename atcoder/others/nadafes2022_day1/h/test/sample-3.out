5
2
2
7
